exasol_script_languages_developer_sandbox/lib/ansible/ansible_repository.py

```diff
@@ -0,0 +1,39 @@
+import shutil
+from pathlib import Path
+
+import jinja2
+
+
+class AnsibleRepository:
+
+    def copy_to(self, target: Path) -> None:
+        pass
+
+
+class AnsibleResourceRepository(AnsibleRepository):
+
+    def __init__(self, path: str):
+        self._path = path
+
+    def copy_to(self, target: Path) -> None:
+        loader = jinja2.PackageLoader("exasol_script_languages_developer_sandbox", package_path=self._path)
+        env = jinja2.Environment(loader=loader, autoescape=jinja2.select_autoescape(), keep_trailing_newline=True)
+        for ansible_file in loader.list_templates():
+            t = env.get_template(ansible_file)
+            content = t.render()
```

Comment:
You use jinka2 to load the file, but call render without variables. What is the intention? And if this includes Ansible files, this might be dangerous, because Ansible use Jinja itself.

Reply:
Fixed